### ISSUE TITLE
Change behaviour with @import rules in css files

### ISSUE THREAD

**Issue by sthag** [abagshaw/brackets-minifier#12]
I have @import rules in my css files.
If I manually or automatically save minified versions the @import rule is left out.

Example test.css:
```
@charset "UTF-8";
@import url(other/normalize.min.css);

/**
**/

body {
	background: black;
}
```

Result test.min.css:
`@charset "UTF-8";body{background:#000}`

Is it possible to change this behaviour? I'd like to have the @import rule just copied over.
Thanks for the extension.
Best regards
sthag

**Comment by abagshaw** [comment]
Hi @sthag, yes it it certainly possible to change this. For some reason I think CleanCSS was trying to resolve the URL and place the contents of the imported file in there - but there is an option to prevent resolving imports. I've just tested it and it does work - I'll try to push an update sometime this evening when I have a chance. Thanks!

**Comment by sthag** [comment]
Hey that is great news. Thank you very much for your quick action and reply. This greatly improves the extension for me. :+1:

**Comment by abagshaw** [comment]
I've pushed the 1.1.6 update to the brackets registry (you should be able to update in brackets) and the problem should be fixed with f971982, if there are any further problems please tell me. Thanks!
